.changeset/clean-cameras-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': minor
+---
+
+Fixed a bug where UpdateData did not recognize optional, dot-separated string fields
```

Comment:
If this goes straight into release notes, I'd suggest `UpdateData` as a literal, and a period.  Otherwise LGTM, thanks!